CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ set(CMAKE_INSTALL_PREFIX "/opt/rocm" CACHE PATH "Install path prefix, prepended
 list(APPEND CMAKE_MODULE_PATH
     ${CMAKE_CURRENT_SOURCE_DIR}/cmake
     ${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules
-    ${HIP_PATH}/cmake /opt/rocm/hip/cmake # FindHIP.cmake
+    ${HIP_PATH}/cmake ${CMAKE_INSTALL_PREFIX}/hip/cmake # FindHIP.cmake
```

Comment:
This should not be CMAKE_INSTALL_PREFIX. We can install the library to separate location other than the rocm location.

Reply:
Will make the changes as discussed in rocPRIM review and will add ROCM_PATH.